packages/components/src/templates/next/layouts/Collection/utils/sortCollectionItems.ts

```diff
@@ -0,0 +1,96 @@
+import type { GetCollectionItemsProps } from "./getCollectionItems"
+import type { AllCardProps } from "~/interfaces"
+
+export type SortableCardProps = AllCardProps & {
+  rawDate?: Date
+}
+
+export interface SortCollectionItemsProps
+  extends Pick<GetCollectionItemsProps, "sortBy" | "sortDirection"> {
+  items: SortableCardProps[]
+}
+
+// Sort by last updated date, tiebreaker by title
+const sortCollectionItemsByDate = ({
+  items,
+  sortDirection = "desc",
+}: Omit<SortCollectionItemsProps, "sortBy">) => {
+  return items.sort((a, b) => {
+    // If both items have no dates, sort by title ascending as a tiebreaker
+    if (a.rawDate === undefined && b.rawDate === undefined) {
+      return a.title.localeCompare(b.title)
```

Comment:
for localeCompare, there might be options to configure iirc

like sensitivity, and numeric

e.g. what if the items start with numbers "1 Item One", "2 Item Two" etc.

Reply:
updated in https://github.com/opengovsg/isomer/pull/1212/commits/7d23cf61368a2abe30354d2baf70b31c79dac01d